rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -967,7 +967,7 @@ error_code cellGemGetCameraState(vm::ptr<CellGemCameraState> camera_state)
 {
 	cellGem.todo("cellGemGetCameraState(camera_state=0x%x)", camera_state);
 
-	auto& gem = g_fxo->get<gem_config>();
+	[[maybe_unused]] auto& gem = g_fxo->get<gem_config>();
```

Comment:
why not just remove it? Is there some hidden stuff going on?

Reply:
gem_config is a named thread so I let it be in case it needs to be created when this api is called? It's not doing nothing.

Reply:
To be clear, I still think there should be the error, just can't find proof.